### PR TITLE
Doc utp for ga release 1.0

### PR DESCRIPTION
**Pull request by johann-r-unity**
**Select the type of change:**

Mostly clean up for UTP Release 1.0

### CODE REVIEW

File: transport/event-consumption.md [hunk 1]
```diff
@@ -54,8 +54,5 @@ while ((c = m_Driver.Accept()) != default(NetworkConnection))
 
 Reviewing this table, note the following:
 
-* The `Connect` event is only available if the `NetworkDriver` is **not** `Listening`.
-  
-  To receive any `Connect` events on a `NetworkDriver` that is in the `Listening` state, call the special function `Accept` just as we did in the *Creating a Server* section in the [Creating a minimal client and server](workflow-client-server.md) workflow page.
+*  To receive any `Connect` events on a `NetworkDriver` that is in the `Listening` state, call the special function `Accept` just as we did in the *Creating a Server* section in the [Creating a minimal client and server](workflow-client-server.md) workflow page.
 
-* If you call `Disconnect` on a `NetworkConnection`, this will not trigger an event inside your own driver.
```
Comment (simon-lemay-unity) [inline transport/event-consumption.md:61]: Why was this removed? I believe it is accurate.

File: transport_versioned_docs/version-1.0.0/pipelines-usage.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ id: pipelines
 title: Use pipelines
 ---
 
-Pipelines are a feature which offers layers of functionality on top of the default socket implementation behaviour. In the case of the UDP socket this makes it possible to have additional functionality on top of the standard unreliable datagram, such as Quality of Service features like sequencing, reliability, fragmentation and so on. This could work with any type of socket interface which has been implemented for use in the driver.
+Pipelines are a feature which offers layers of functionality on top of the default socket implementation behaviour. In the case of the UDP socket this makes it possible to have additional functionality on top of the standard unreliable datagram, such as Quality of Service features like sequencing, reliability, fragmentation and so on. This could work with any type of UDP socket interface which has been implemented for use in the driver.
```
Comment (simon-lemay-unity) [inline transport_versioned_docs/version-1.0.0/pipelines-usage.md:6]: Technically it would also work with any non-UDP socket interface. Considering that we currently only support one interface, and that creating custom ones is a very advanced functionality, I think it's simpler to just drop that last sentence.

**Comment by simon-lemay-unity** [comment]
The PR is not very consistent about what changes go to the `transport` directory, and which ones go in `transport_versioned_docs/version-1.0.0`. My understanding is that at this point, both versions should be identical.

File: transport/event-consumption.md [hunk 3]
```diff
@@ -54,8 +54,5 @@ while ((c = m_Driver.Accept()) != default(NetworkConnection))
 
 Reviewing this table, note the following:
 
-* The `Connect` event is only available if the `NetworkDriver` is **not** `Listening`.
-  
-  To receive any `Connect` events on a `NetworkDriver` that is in the `Listening` state, call the special function `Accept` just as we did in the *Creating a Server* section in the [Creating a minimal client and server](workflow-client-server.md) workflow page.
+*  To receive any `Connect` events on a `NetworkDriver` that is in the `Listening` state, call the special function `Accept` just as we did in the *Creating a Server* section in the [Creating a minimal client and server](workflow-client-server.md) workflow page.
```
[the remark this answers is not in the document]
Comment (johann-r-unity) [reply transport/event-consumption.md:57]: Yes, please let me know.

File: transport/event-consumption.md (hunk 1 above)
Comment (johann-r-unity) [reply transport/event-consumption.md:61]: It is, it was just a duplicate from what was already stated in the table a few lines above.

File: transport_versioned_docs/version-1.0.0/pipelines-usage.md (hunk 2 above)
Comment (johann-r-unity) [reply transport_versioned_docs/version-1.0.0/pipelines-usage.md]: Removed.

File: transport/event-consumption.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (simon-lemay-unity) [reply transport/event-consumption.md:57]: Can confirm that `Connect` is only generated upon receiving a `ConnectionAccept` message from a server. So only clients can receive these events.

**Comment by johann-r-unity** [comment]
> The PR is not very consistent about what changes go to the `transport` directory, and which ones go in `transport_versioned_docs/version-1.0.0`. My understanding is that at this point, both versions should be identical.

Good point. I ported the modification from transport to transport_versioned_doc\version-1.0.0 

@sara-omeilia-unity , what is the correct way to modify the docs here ? 
Is the `transport` folder simply the most up to date docs ?

**Review by s-omeilia-unity — APPROVED** [review]
